index.js

```diff
@@ -116,7 +104,29 @@ async function transact (wallet) {
 }
 
 ;(async () => {
+  chainId = await lookupChainId(httpProviderUrl)
```

Comment:
From running last week i realized that, if the txpool api is not enabled in the node, this crashes, but its not easy to know what is wrong, so if the txpool_status returns not available, maybe throw that error, so that people know to add txpool into their websocket api lists in the node

Reply:
I recently made the error handling a lot more clear. See lines 114&129 in utils.js

The error comes directly from go-quai now and displays properly :)